server/storage/files/S3Storage.ts

```diff
@@ -86,10 +90,10 @@ export default class S3Storage extends BaseStorage {
     acl,
   }: {
     body: S3.Body;
-    contentLength: number;
-    contentType: string;
+    contentLength?: number;
```

Comment:
Tempting to say these should remain as required, even if not used in all implementations